pkgs/top-level/linux-kernels.nix

```diff
@@ -0,0 +1,525 @@
+{ pkgs
```

Comment:
I don't want to bike shed too much, but this should probably go in `pkgs/os-specific/linux`.

Then again, we have darwin packages. Probably relevant for a larger discussion.

cc @vcunat @Mic92 @NeQuissimus (and a bunch of others)

Reply:
I think `top-level` is a good place for package sets.

Reply:
meh, I guess I don't feel strongly about it, and it seems that the current paradigm is to use `top-level`. I'll resolve this.